Jenkins_jobs/CreateNewNode.groovy

```diff
@@ -26,14 +26,24 @@ node {
         for (int index = 0; index < machineIPs.length; index++) {
 
             if (nodeHelper.getComputer(machines[index]) != null) {
-                println "Machine(${machines[index]}) already exists."
+                println "Machine: '${machines[index]}' already exists."
             } else {
                 if (Integer.parseInt(machineIPs[index].split("\\.")[0]) == 10) {
                     launcher = new CommandLauncher(Constants.SSH_COMMAND + "${machineIPs[index]} " + Constants.WGET_SLAVE_JAR);
                     remoteFS = Constants.REMOTE_FS;
                 } else if (machines[index].contains("win")) {
                     launcher = new JNLPLauncher("", "", new jenkins.slaves.RemotingWorkDirSettings(false, "", "remoting", false));
                     remoteFS  = Constants.WIN_REMOTE_FS;
+                } else if (machines[index].contains("osx")){
```

Comment:
Since this case is very similar to the `else` case, can they be combined and add an `if` around the `remoteFS` instead?